website/client/components/tasks/spells.vue

```diff
@@ -236,35 +237,39 @@ export default {
       this.spell = spell;
 
       if (spell.target === 'self') {
-        this.castEnd(null, 'self');
+        this.castEnd(null, spell.target);
```

Comment:
@TheHollidayInn  This change from `'self''` to `spell.target` (and the similar changes below) aren't strictly necessary but make the code a bit clearer I believe, since it makes it obvious you're just passing the target through to the next stage. It also avoids a chance of a future bug if code is copy-pasted to make new code for a similar purpose -- no need to change '`self'` in two places.